### PR TITLE
WIP: Debian / compat with msmtpd

### PR DESCRIPTION
**Pull request by drzraf**
Abilty to suffix the sbin/ binary and use update-alternative so that smailq can be installed alongside msmtpd (msmtp-mta on Debian)

Related: https://salsa.debian.org/kolter/msmtp/-/merge_requests/15

### CODE REVIEW

**Comment by shuber2** [comment]
Looks good, but could you rephrase the commit message for a more friendly first line?

**Comment by drzraf** [comment]
done

**Comment by shuber2** [comment]
What do you think about this commit message?
https://github.com/shuber2/smailq/commit/2e355565092fb8b1121dd4bb335c9ae1c69ba9df

**Comment by drzraf** [comment]
Fine for me, of course (Feel free to adapt as you wish)

**Comment by shuber2** [comment]
Merged.
